estudios-de-cultura-maya.csl

```diff
@@ -0,0 +1,346 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="es-MX">
+  <info>
+    <title>Estudios de Cultura Maya</title>
```

Comment:
This should be `<title>Estudios de Cultura Maya (Spanish - Mexico)</title>` (add language descriptor to all styles fixed to non-English locales).